### ISSUE TITLE
Narrator is silent when user selects a NavigationViewItem

### ISSUE THREAD

**Issue by YuliKl** [microsoft/microsoft-ui-xaml#3296]
**Describe the bug**

Narrator users get no audible feedback to confirm their selection. This feels particularly problematic because selecting a nav item likely results in a page navigation and a significant change to the app content. Sighted users get visual feedback that something happened but blind users are left, so to speak, in the dark.

**Steps to reproduce the bug**

1. Turn on Narrator and open Xaml Controls Gallery
2. Move focus to Basic Input item
3. Press Space or Enter to active it

**Expected behavior**

User hears audible feedback along the lines of "Basic Input, 3 of 14, selected, expanded". This may be unnecessary verbose and simply announcing "selected" may suffice.

**Comment by StephenLPeters** [comment]
I think we need to raise a UIA Selection event https://docs.microsoft.com/en-us/windows/win32/winauto/uiauto-implementingselectionitem

**Comment by chingucoding** [comment]
This does not repro with the latest debug build of XCG and the MUXControlsTestApp. I think this was fixed with #2692 but just not serviced to the store version of the XCG.

**Comment by YuliKl** [comment]
Closing based on @chingucoding's last comment

**Comment by liangxy06** [comment]
Checked the issue still exists in the latest 1.2.20.0 version

**Steps to reproduce the bug:**
Turn on Narrator and open Xaml Controls Gallery
Move focus to the bottom Settings item
Press Space or Enter to active it

**Expected behavior:**
User hears audible feedback along the lines of "Settings, 1 of 1, selected". 

**Actual behavior:**
Narrator does not announce feedback, and focus goes to "Back" button instead

**Comment by StephenLPeters** [comment]
Seems to work fine in the Left Expanded mode. I do notice that in the minimal mode the pane is collapsed on invocation causing narrator to move to a different element and not reading the settings selection, is that what you are referring to?
